server.js

```diff
@@ -566,20 +566,26 @@ app.post(pathPrefix + '_changes', jsonParser, changesHander);
 var writeHeaders = function(req, res, headers, redirectHumans) {
   res.oldWriteHead = res.writeHead;
   res.writeHead = function(_statusCode, _headers) {
-    // hardcode this so we never show the basic auth prompt
+    // hardcode this so we do not show the basic auth prompt by default
     res.setHeader('WWW-Authenticate', 'Cookie');
```

Comment:
Can we move this line down into the `isClientHuman` block so we're not writing the header twice?